### DISCUSSION TITLE
I found a potential flaw in the score system

### DISCUSSION THREAD

**Discussion by senpai-haze-sama** [ppy/osu#26890] [Ideas]
if you submit a score on lazer, and then play the same map on stable, you get the ranked score twice instead of just the difference

**Comment by Rekunan** [comment]
Had this happen to me:
Lazer: https://osu.ppy.sh/scores/2278574208
Stable: https://osu.ppy.sh/scores/osu/4578820731

I forgot that lazer can't give medals, so I went on stable to get the 7* pass medal, and then now I have 2 saved scores for NM.

**Reply by Rekunan** [reply]
Actually thinking about it more, it might be because stable scores appear with the classic mod in lazer, so in reality they aren't both NM scores, but rather 1 CL and 1 NM.

**Reply by Rekunan** [reply]
I'm leaning way more towards this answer after finding out that Bathbot shows both scores:
![image](https://github.com/ppy/osu/assets/24501632/83817ccd-cf72-4e4b-abfd-83699e310b7b)
@ppy/team-client pinging for a confirmation on this

**Reply by bdach** [reply]
No, that's not how that works. Mod combinations do not factor into ranked score processing.

The reason why this is happening is that stable score processing is running parallel to lazer's, and stable's sees only stable scores while lazer sees all of them. This will only resolve when legacy score processing gets turned off.

**Comment by respektive** [comment]
I think the issue is that when stable uploads a score, the server-side part that's supposed to check if there's already a score on the map to calculate the ranked score difference can't see the lazer scores, because they're in a different database table. This will probably be fixed once stable uses the new statistics processors. Albeit, the ranked score totals will be wrong now and need to be recalculated, but there are also a few more other issues that should be addressed before recalculating anything. See: https://github.com/ppy/osu/discussions/27007

**Reply by peppy** [reply]
It is precisely as you say. We'll recalculate totals once this is resolved.
